src/rez/tests/test_shells.py

```diff
@@ -331,8 +333,9 @@ def _print(value):
 
         # We are wrapping all variable outputs in quotes in order to make sure
         # our shell isn't interpreting our output as instructions when echoing
-        # it but this means we need to wrap our expected output as well.
-        expected_output = ['"{}"'.format(o) for o in expected_output]
+        # it but this means we need to wrap our expected output as well. Only
+        # exception is empty string, which is just passed through.
+        expected_output = ['"{}"'.format(o) if o else o for o in expected_output]
```

Comment:
This was necessary since the interpretation of empty string in quotes would not match throughout the shells.
Not sure if this is correct but the data itself is correct (empty) and this is what I wanted to compare here.